app/permissions/team.rb

```diff
@@ -0,0 +1,119 @@
+Canaid::Permissions.register_for(Team) do
+  # view projects, view protocols
+  # leave team, view team users (ATWHO)
+  # view samples, export samples
+  # view repositories, view repository, export repository rows
+  can :read_team do |user, team|
+    user.is_member_of_team?(team)
+  end
+
+  # edit team name, edit team description
+  can :update_team do |user, team|
+    user.is_admin_of_team?(team)
+  end
+
+  # invite user to team, change user's role, remove user from team
+  can :manage_user_team do |user, team|
+    user.is_admin_of_team?(team)
+  end
+
+  # create project
+  can :create_project do |user, team|
+    user.is_normal_user_or_admin_of_team?(team)
+  end
+
+  # create protocol in repository, import protocol to repository
+  can :create_protocol do |user, team|
+    user.is_normal_user_or_admin_of_team?(team)
+  end
+
+  # create, import, edit, delete samples
+  can :manage_samples do |user, team|
+    user.is_normal_user_or_admin_of_team?(team)
+  end
+
+  # create custom field
+  # create, update, delete sample type or sample group
+  can :manage_sample_elements do |user, team|
+    user.is_normal_user_or_admin_of_team?(team)
+  end
+
+  # create, copy repository
+  can :manage_repository do |user, team|
```

Comment:
Probably better would be to keep `can_create_repositories?`. Manage relates more to editing, updating it right?